ush/build.sh

```diff
@@ -47,8 +114,15 @@ CMAKE_OPTS+=" -DGSI_MODE=$GSI_MODE -DENKF_MODE=${ENKF_MODE}"
 mkdir -p $BUILD_DIR && cd $BUILD_DIR
 
 # Configure, build, install
-cmake $CMAKE_OPTS $DIR_ROOT
-make -j ${BUILD_JOBS:-8} VERBOSE=${BUILD_VERBOSE:-}
-make install
+#     specifit options for 3DRTMA
+if [[ -v SAVELOG && ${SAVELOG} =~ [yYtT] ]] ; then
```

Comment:
Nice feature but a similar result can be achieved with the original `build.sh` via `build.sh > log.build 2>&1`.  `log.build` contains both the build and install information.  `log.build` will be in `/ush`, not `/build`.